int64 joystick_vertical
int64 joystick_horizontal